code/modules/antagonists/blob/blob_abilities.dm

```diff
@@ -248,14 +247,14 @@
 
 		if (!owner.tutorial)
 			//do a little "blobsplosion"
+		//sound and sound code additions by retkid#9135, coding assitance by pali#0439 and Paul#7955
```

Comment:
why comment this here?
also would better go in the PR, we track diff changes for the purposes of license-compliant attribution